utils/goutils_test.go

```diff
@@ -41,71 +41,47 @@ rsc.io/sampler:v1.3.0
 
 func TestGetProjectRoot(t *testing.T) {
 	wd, err := os.Getwd()
-	if err != nil {
-		t.Error(err)
-	}
-	defer os.Chdir(wd)
+	assert.NoError(t, err)
+	defer func() {
+		assert.NoError(t, os.Chdir(wd))
+	}()
 
 	// CD into a directory with a go.mod file.
 	projectRoot := filepath.Join("testdata", "project")
-	err = os.Chdir(projectRoot)
-	if err != nil {
-		t.Error(err)
-	}
+	assert.NoError(t, os.Chdir(projectRoot))
 
 	// Make projectRoot an absolute path.
 	projectRoot, err = os.Getwd()
-	if err != nil {
-		t.Error(err)
-	}
+	assert.NoError(t, err)
 
 	// Get the project root.
 	root, err := GetProjectRoot()
-	if err != nil {
-		t.Error(err)
-	}
-	if root != projectRoot {
-		t.Error("Expecting", projectRoot, "got:", root)
-	}
+	assert.NoError(t, err)
+	assert.Equal(t, projectRoot, root)
 
 	// CD back to the original directory.
-	if err := os.Chdir(wd); err != nil {
-		t.Error(err)
+	if !assert.NoError(t, os.Chdir(wd)) {
+		return
 	}
```

Comment:
I think that flat code in tests is easier to understand
```suggestion
	assert.NoError(t, os.Chdir(wd))
```

Reply:
It is easier to understand, but if changing dir failed here the rest of the test is irrelevant. This is used to fail the test fast